apps/frontend/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import vector from '../../assets/icons/vector.svg';
+import clsx from 'clsx';
+
+interface TagProps {
+  label: string;
+  isSelected?: boolean;
+}
+
+export const Tag = ({ label, isSelected }: TagProps) => {
+    const [selected, setSelected] = React.useState<boolean>(isSelected || false);
+
+  const commonClasses = 'rounded-[2.5rem] font-gellix text-[1.25rem] leading-5 p-[1.875rem]';
```

Comment:
Just a headsup, but not really a problem
`text-[1.25rem]` is tailwind `text-xl`